profiles/fscloud/README.md

```diff
@@ -74,12 +73,13 @@ This rule is ignored because the module achieves the same resiliency as cross-re
 
 | Name | Description |
 |------|-------------|
+| <a name="output_buckets"></a> [buckets](#output\_buckets) | Bucket module output |
 | <a name="output_cos_instance_guid"></a> [cos\_instance\_guid](#output\_cos\_instance\_guid) | The GUID of the Cloud Object Storage Instance where the buckets are created |
 | <a name="output_cos_instance_id"></a> [cos\_instance\_id](#output\_cos\_instance\_id) | The ID of the Cloud Object Storage Instance where the buckets are created |
 | <a name="output_primary_bucket_id"></a> [primary\_bucket\_id](#output\_primary\_bucket\_id) | Primary Bucket id |
 | <a name="output_primary_bucket_name"></a> [primary\_bucket\_name](#output\_primary\_bucket\_name) | Primary Bucket Name |
 | <a name="output_resource_group_id"></a> [resource\_group\_id](#output\_resource\_group\_id) | Resource Group ID |
 | <a name="output_s3_endpoint_private"></a> [s3\_endpoint\_private](#output\_s3\_endpoint\_private) | S3 private endpoint |
 | <a name="output_secondary_bucket_id"></a> [secondary\_bucket\_id](#output\_secondary\_bucket\_id) | Secondary Bucket id |
-| <a name="output_secondary_bucket_name"></a> [secondary\_bucket\_name](#output\_secondary\_bucket\_name) | Primary Bucket Name |
+| <a name="output_secondary_bucket_name"></a> [secondary\_bucket\_name](#output\_secondary\_bucket\_name) | Secondary Bucket Name |
```

Comment:
Not a big deal, but bucket doesn't need to be capitalized here, just above, or throughout the readme files because it's not a proper noun or part of a offering name.